Paintroid/src/main/java/org/catrobat/paintroid/UserPreferences.java

```diff
@@ -15,6 +16,17 @@ public boolean getBoolean(String tag, boolean defaultReturnValue) {
 		return preferences.getBoolean(SHOW_LIKE_US_DIALOG_SHARED_PREFERENCES_TAG, false);
 	}
 
+	public int getInt() {
+		return preferences.getInt(IMAGE_NUMBER_SHARED_PREFERENCES_TAG, 0);
+	}
+
+	public void setInt(String tag, int value) {
+		preferences
+			.edit()
+			.putInt(IMAGE_NUMBER_SHARED_PREFERENCES_TAG, value)
+			.apply();
+	}
```

Comment:
Please change to getter and setter for the corresponding variable not just Int.

Also do the same for getBoolean/setBoolean.